example/reinforcement-learning/a3c/a3c.py

```diff
@@ -153,8 +153,11 @@ def train():
                 module.forward(batch, is_train=True)
 
                 pi = module.get_outputs()[1]
-                h = args.beta*(mx.nd.log(pi+1e-6)+1)
-                module.backward([mx.nd.array(adv), h])
+                h = -args.beta*(mx.nd.log(pi+1e-7)*pi)
+                out_acts = np.amax(pi.asnumpy(), 1)
```

Comment:
I looked at the paper again and looks like should use act from https://github.com/dmlc/mxnet/pull/5473/files#diff-03ce9792a200a63897dce33fd2967c39R139 instead of amax?